update readme in nextjs testing (#54322)
vercel/next.js: docs/03-pages/01-building-your-application/05-optimizing/10-testing.mdx

```

> Then, inside `jest.setup.js`, add the following import:

Reviewer: ```suggestion
Then, inside `jest.setup.js`, add the following import:

```

```

Then, inside `jest.setup.js`, add the following import:
```js filename="jest.setup.js"

> import '@testing-library/jest-dom/extend-expect'

Reviewer: ```suggestion
import '@testing-library/jest-dom'
```

```

> If you are using `@testing-library/jest-dom` release 6 and later, use the code below instead. As per the latest `@testing-library/jest-dom` [release (v6.0)](https://github.com/testing-library/jest-dom/releases/tag/v6.0.0) the `extend-expect` was removed.

Reviewer: ```suggestion
> [`extend-expect` was removed in `v6.0`](https://github.com/testing-library/jest-dom/releases/tag/v6.0.0), so if you are using `@testing-library/jest-dom` before version 6, you will need to import `@testing-library/jest-dom/extend-expect` instead.
```

> ```js filename="jest.setup.js"

Reviewer: ```suggestion
```

> import '@testing-library/jest-dom

Reviewer: ```suggestion
```

> ```

Reviewer: ```suggestion
```

> 

Reviewer: ```suggestion
```
